packages/amplify-graphql-api-construct/package.json

```diff
@@ -18,17 +18,17 @@
     "test": "jest",
     "test-watch": "jest --watch",
     "extract-api": "ts-node --project extract-api.tsconfig.json ../../scripts/extract-api.ts",
-    "update-readme": "cp README_HEAD.md README.md && jsii && jsii-docgen --readme --output README.md"
+    "update-readme": "cp README_HEAD.md README.md && jsii && jsii-docgen --readme --output README.md",
+    "prepare": "jsii && git add .jsii && git commit -m \"chore: update .jsii assembly\" --no-verify"
```

Comment:
Should this be `prepublishOnly`? I'm not much of an expert on the life cycle scripts though.

Reply:
Yeah, I think that might be right. Since prepare could possibly run on client machines, if I'm reading the docs correctly.

Reply:
Updated to use that now, and re-running my tag release.